### PR TITLE
[WIP] ENH Add CodeCov upload of results

### PR DESCRIPTION
**Pull request by mike-wendt**
Working on integrating CodeCov results with this repo, see rapidsai/ops#42

### CODE REVIEW

File: ci/gpu/build.sh [hunk 1]
```diff
@@ -95,3 +95,6 @@ conda install -c conda-forge -y feather-format
 logger "Python py.test for cuDF..."
 cd $WORKSPACE/python
 py.test --cache-clear --junitxml=${WORKSPACE}/junit-cudf.xml -v --cov-config=.coveragerc --cov=cudf --cov-report=xml:${WORKSPACE}/cudf-coverage.xml --cov-report term
+
+logger "Upload CodeCov results..."
+curl -s https://codecov.io/bash | bash -s -- -c -F aFlag -f ${WORKSPACE}/cudf-coverage.xml
```
Comment (rlratzel) [inline ci/gpu/build.sh]: I don't feel too strongly about this, but there seems to be many others that have problems with "curl bash piping". I know CodeCov documents this as the way to upload results, but they also released a Python package (available from both `pip` and `conda`) so you don't have to do a curl|bash

https://github.com/codecov/codecov-python

**Comment by mike-wendt** [comment]
rerun tests

**Comment by mike-wendt** [comment]
rerun tests

**Comment by kkraus14** [comment]
Resolved by #2171

**Pull request closed without merging**